db/schema.rb

```diff
@@ -22,12 +22,34 @@
     t.text     "summary"
     t.integer  "parent_id"
     t.integer  "children_count"
-    t.datetime "created_at",                     null: false
-    t.datetime "updated_at",                     null: false
-    t.boolean  "placeholder",    default: false, null: false
+    t.datetime "created_at",     null: false
+    t.datetime "updated_at",     null: false
     t.index ["slug"], name: "index_categories_on_slug", using: :btree
   end
 
+  create_table "data_bundles", force: :cascade do |t|
+    t.integer  "node_id"
+    t.string   "type"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["node_id"], name: "index_data_bundles_on_node_id", using: :btree
+  end
+
+  create_table "data_public_holidays", force: :cascade do |t|
+    t.string   "name"
+    t.date     "date"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
The data_bundles and data_public_holidays tables are not created in this PR. Should they be here?